File: csharp/src/JewelleryTypes/Pendant.cs
using NecklaceRefactoringKata.Enums;

namespace NecklaceRefactoringKata.JewelleryTypes
{
    public record Pendant(Jewel Stone) : JewelleryBase(Stone)
    {
        public override bool IsSmall()
        {
            return true;
        }
    }
}

File: java/kata/src/main/java/necklaceRefactoringKata/jewellery/Pendant.java
package necklaceRefactoringKata.jewellery;

public final class Pendant extends Jewellery {

    public Pendant(Jewel stone) {
        super(stone);
    }

    @Override
    public boolean isSmall() {
        return true;
    }

    @Override
    public String toString() {
        return "Pendant{" +
                "stone=" + stone +
                '}';
    }
}
